pandas/core/algorithms.py

```diff
@@ -1110,10 +1106,7 @@ def compute(self, method):
         n = self.n
         dtype = self.obj.dtype
         if not self.is_valid_dtype_n_method(dtype):
-            raise TypeError(
-                "Cannot use method '{method}' with "
-                "dtype {dtype}".format(method=method, dtype=dtype)
-            )
+            raise TypeError(f"Cannot use method '{method}' with " f"dtype {dtype}")
```

Comment:
This can just be one string - can you fix implicit concatenation?

Reply:
Thanks @WillAyd - sorry, my bad. It's fixed now.